src/inlay-hints.ts

```diff
@@ -53,11 +53,7 @@ class InlayHintsFeature implements vscodelc.StaticFeature {
     const serverCapabilities: vscodelc.ServerCapabilities&
         {clangdInlayHintsProvider?: boolean, inlayHintProvider?: any} =
         capabilities;
-    // If the clangd server supports LSP 3.17 inlay hints, these are handled by
-    // the vscode-languageclient library - don't send custom requests too!
-    if (!serverCapabilities.clangdInlayHintsProvider ||
-        serverCapabilities.inlayHintProvider)
-      return;
+
     if (!this.commandRegistered) {
```

Comment:
we actually still need to govern entry here. we shouldn't register the command if neither inlay hints extension nor standard protocol are advertised by the server.

Reply:
Good point. We should even disable the 'toggle' command in VSCode command palette if users use a clangd without inlay-hints support (v13 or before).